sdk/config/crd/replacements/release_name_validation.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: release-name-validation
+  namespace: release-name-validation
+data:
+  pattern: '^(((aws|azure|vsphere|cloud-director|eks)-)|v)(0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)(-[\.0-9a-zA-Z]*)?$'
```

Comment:
Release CR name validation.

P.S. This ConfigMap is just used for name pattern patching when building the Release CRD. The ConfigMap itself is not output to resulting manifests.